scene/resources/convex_polygon_shape_2d.h

```diff
@@ -41,6 +41,8 @@ class ConvexPolygonShape2D : public Shape2D {
 
 protected:
 	static void _bind_methods();
+	static bool left_test(const Vector2 &p1, const Vector2 &p2, const Vector2 &p3);
+	static bool is_convex(const Vector<Vector2> &p_points);
```

Comment:
Should be private and not protected I think (or might work also as just static helpers in the .cpp not declared in the header).